test/remote/archiver.go

```diff
@@ -53,8 +53,9 @@ func CreateDriverArchive(archiveName, pkgPath, binPath string) (string, error) {
 	return filepath.Join(dir, archiveName), nil
 }
 
-func setupBinaries(tarDir, pkgPath, binPath string) error {
+func setupBinaries(architecture, tarDir, pkgPath, binPath string) error {
 	klog.V(4).Infof("Making binaries and copying to temp dir...")
+	os.Setenv("ARCH", architecture)
 	out, err := exec.Command("make", "-C", pkgPath).CombinedOutput()
```

Comment:
I think you can also just call "make -C pkgPath ARCH=<architecture>", rather than setting an environment variable?

Reply:
done